api_tests/dry/multiple_peers.ts

```diff
@@ -1,207 +1,85 @@
-import { expect, request } from "chai";
+// These are stateless tests -- they don't require any state of the cnd and they don't change it
+// They are mostly about checking invalid request responses
+// These test do not use the sdk so that we can test edge cases
+import { threeActorTest } from "../lib_sdk/actor_test";
+import { expect } from "chai";
 import "chai/register-should";
-import { ethers } from "ethers";
-import { EmbeddedRepresentationSubEntity } from "../gen/siren";
-import { Actor } from "../lib/actor";
 import "../lib/setup_chai";
-
-(async () => {
-    const alpha = {
-        ledger: {
-            name: "bitcoin",
-            network: "regtest",
-        },
-        asset: {
-            name: "bitcoin",
-            quantity: {
-                bob: "100000000",
-                charlie: "200000000",
-            },
-        },
-        expiry: new Date("2080-06-11T23:00:00Z").getTime() / 1000,
-    };
-
-    const beta = {
-        ledger: {
-            name: "ethereum",
-            chain_id: 17,
-        },
-        asset: {
-            name: "ether",
-            quantity: {
-                bob: ethers.utils.parseEther("10").toString(),
-                charlie: ethers.utils.parseEther("20").toString(),
-            },
-        },
-        expiry: new Date("2080-06-11T13:00:00Z").getTime() / 1000,
+import { SwapDetails } from "comit-sdk/dist/src/cnd";
+import {createDefaultSwapRequest} from "../lib_sdk/utils";
```

Comment:
Does not look like it went through prettier.

Reply:
Yes, missed it. done now.